src/main/java/org/embulk/input/facebook_ads_insights/Client.java

```diff
@@ -56,9 +56,16 @@ public Client(final PluginTask pluginTask)
             default: throw new IllegalArgumentException();
         }
         logger.info(adReportRun.getRawResponse());
+        int asyncLoopCount = 0;
         while (adReportRun.fetch().getFieldAsyncPercentCompletion() != 100) {
             logger.info(adReportRun.getRawResponse());
             Thread.sleep(ASYNC_SLEEP_TIME);
+            if (adReportRun.getFieldAsyncStatus().equals("Job Skipped")) {
+                throw new RuntimeException("Transfer was aborted because the AsyncStatus is \"Job Skipped\"");
+            }
+            if (++asyncLoopCount >= 300) {
+                throw new RuntimeException("Transfer was aborted because the AsyncStatus remains \"Job Not Started\"");
```

Comment:
すごい細かいんですけど、
"Transfer was aborted because the AsyncStatus remains \"Job Not Started\""
remains Job Not Startedって書いてしまうと、もしかしたらExecuting中でタイムアウトする場合もあるのかなと思いまして。
後々何かあったときに別の人が調査する際に正しいエラーメッセージの方がいい気がするので、所定時間が経ったけど終わってない的な意味の方がいいと思いました。

Reply:
ありがとうございます。
エラーメッセージを
```
async was aborted because the AsyncStatus is \"Job Skipped\"
async was aborted because the number of retries exceeded the limit
```
に修正しました。